src/host/ut_host/TextBufferTests.cpp

```diff
@@ -2018,7 +2018,7 @@ void TextBufferTests::TestBurrito()
     VERIFY_IS_FALSE(afterBurritoIter);
 }
 
-void TextBufferTests::WriteLinesToBuffer(std::vector<std::wstring> text, TextBuffer& buffer)
+void TextBufferTests::WriteLinesToBuffer(const std::vector<std::wstring> text, TextBuffer& buffer)

```

Comment:
you missed an `&` as well.

Reply:
this .. still? seems to be missing an `&`?

Reply:
idk how I missed this comment _twice_!